instruments/srs/srsctc100.py

```diff
@@ -235,7 +235,8 @@ def get_log_point(self, which='next', units=None):
                     'getLog.xy {}, {}'.format(self._chan_name, which)
                 ).split(',')
             ]
-            return u.Quantity(point[0], 'ms'), u.Quantity(point[1], units)
+            return u.Quantity(float(point[0]), 'ms'), \
+                   u.Quantity(float(point[1]), units)
```

Comment:
Not a fan of `\`, would rather see this return value wrapped in `( )` so that its not needed

Reply:
Fixed.